packages/web-components/src/components/va-accordion/va-accordion.tsx

```diff
@@ -8,8 +8,16 @@ import {
   Prop,
   State,
   h,
+  forceUpdate
 } from '@stencil/core';
 import { getSlottedNodes } from '../../utils/utils';
+import i18next from 'i18next'
+import { Build } from '@stencil/core';
+
+if (Build.isTesting) {
+  // Make i18next.t() return the key instead of the value
+  i18next.init({ lng: 'cimode' });
+}
```

Comment:
I'm not a huge fan of this but I still haven't found a way around it that will let us at least have something  in the tests.